model_analyzer/config/generate/manual_model_config_generator.py

```diff
@@ -66,25 +66,27 @@ def _done_walking(self):
            and self._done_walking_max_batch_size()
 
     def _done_walking_configs(self):
-        return len(self._configs) == self._curr_config_index + 1
+        return len(self._configs) == self._curr_config_index
 
     def _done_walking_max_batch_size(self):
         if (self._max_batch_sizes is None or len(self._max_batch_sizes)
-                == self._curr_max_batch_size_index + 1):
+                == self._curr_max_batch_size_index):
             return True
 
-        if self._early_exit_enable and not self._last_results_increased_throughput(
-        ):
+        if self._early_exit_enable \
```

Comment:
are these formatting changes coming from yapf or  something else?

Reply:
No it wasn't. Reverted back to the original format.